WordPress/WordPressShareExtension/ShareViewController.swift

```diff
@@ -3,32 +3,39 @@ import Social
 import WordPressComKit
 
 
-class ShareViewController: SLComposeServiceViewController {
+class ShareViewController: SLComposeServiceViewController
+{
```

Comment:
I believe our standard is same line for curly braces on class/struct definitions in Swift.